src/components/ModDirectory.vue

```diff
@@ -3,7 +3,7 @@
     :on-filepath-change="onModDirectoryChange"
     :pre-filepath-change="preModDirectoryChange"
     :initial-filepath="modDirectory"
-    label="Ultimate Skyrim Directory"
+    label="Select Your Content Folder"
```

Comment:
Hmmm... I'm not sure I'd know what this means if I saw it. Do we want a combination of the 2?
```suggestion
    label="Select your Ultimate Skyrim content directory"
```

Reply:
Actually, this looks a bit gross on the settings page where this is also used 
![image](https://user-images.githubusercontent.com/5766291/131551106-10b56f46-a73c-43bd-8578-966d07546736.png)

Reply:
Maybe just
```suggestion
    label="Ultimate Skyrim Content Folder"
```

Reply:
![image](https://user-images.githubusercontent.com/5766291/131551275-e05b6999-ab70-43f4-a5c1-ebcdcb1f5180.png)

![image](https://user-images.githubusercontent.com/5766291/131551306-d3fefbe9-40e3-46fe-a15e-15a824b519fd.png)